app/assets/javascripts/poller.js

```diff
@@ -5,15 +5,45 @@
     init: function() {
       this.$poller = $('.js-poller');
       this.$messageForm = $('.js-message-form');
+      this.pollingTimeout = 1000 * 60 * 60; // 1 hour
 
       if (this.$poller.length && this.$messageForm.length) {
         this.setLast(this.timestamp());
+        this.visibilitySetup();
         this.start();
+        this.bindEvents();
       }
     },
 
+    visibilitySetup: function() {
+      if (typeof document.hidden !== "undefined") { // Opera 12.10 and Firefox 18 and later support
```

Comment:
Have changed this around a bit, to handle unsupported browsers of the PageVisibility API